eng/Versions.props

```diff
@@ -8,4 +8,21 @@
     <PreReleaseVersionLabel>alpha</PreReleaseVersionLabel>
     <PreReleaseVersionIteration>1</PreReleaseVersionIteration>
   </PropertyGroup>
+
+  <PropertyGroup>
+    <AzureActiveDirectoryIdentityModelExtensionsExternalPackageVersion>7.1.2</AzureActiveDirectoryIdentityModelExtensionsExternalPackageVersion>
+    <CssParserExternalPackageVersionSuffix>20230414.1</CssParserExternalPackageVersionSuffix>
+    <HumanizerCoreExternalPackageVersion>2.14.1</HumanizerCoreExternalPackageVersion>
+    <MSBuildLocatorExternalPackageVersion>1.6.10</MSBuildLocatorExternalPackageVersion>
+    <SpectreConsoleExternalPackageVersion>0.48.0</SpectreConsoleExternalPackageVersion>
+    <XunitAbstractionsExternalPackageVersion>2.0.3</XunitAbstractionsExternalPackageVersion>
+    <XunitExternalPackageVersion>2.4.2</XunitExternalPackageVersion>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <MicrosoftBuildVersion>15.7.179</MicrosoftBuildVersion>
```

Comment:
Personally, I would just use this property and drop the rest.  The versions are all tied and having one property eases maintenance.

Reply:
Done.